pygin/pygin/src/panel.cpp

```diff
@@ -3,6 +3,27 @@
 #include "panel.hpp"
 #include "far_api.hpp"
 #include "py.integer.hpp"
+#include "python.hpp"
+
+#include <ranges>
```

Comment:
Please move it [here](https://github.com/trexinc/evil-programmers/blob/f56297a6324a76d3f5730487c272f2602c67c6f6/pygin/pygin/src/headers.hpp#L40).

Reply:
Maybe copy rather than move? I believe it was always a good thing to see at a glance module's dependencies and not rely on precompiled headers...

`headers.hpp` was updated in 25bec961a4332cba0ae5649d2dd13af0bff6261c.

Reply:
That's a good point, but let's keep the things consistent for now.
Also, we include headers.hpp explicitly anyway.

Reply:
Fixed in 019c2362256ce9cdb41550ea39fa56abcf5bca32.